UIKitDZ/UIKitDZ/EditTextViewController.swift

```diff
@@ -0,0 +1,126 @@
+        
+        return boldSwitch
+    }()
+    private var linePicker: UIPickerView = {
+        var picker = UIPickerView(frame: CGRect(x: 50, y: 500, width: 100, height: 100))
+        picker.tag = 0
+        picker.backgroundColor = .systemBlue
+        return picker
+    }()
+    private var colorPicker: UIPickerView = {
+        var picker = UIPickerView(frame: CGRect(x: 250, y: 500, width: 100, height: 100))
+        picker.tag = 1
+        picker.backgroundColor = .systemGreen
+        return picker
+    }()
+    private var textSizeSlider: UISlider = {
+        let slider = UISlider(frame: CGRect(x: 50, y: 670, width: 310, height: 30))
+        slider.minimumValue = 5
+        slider.value = 30
+        slider.maximumValue = 72
+        return slider
+    }()
+    
+    private var colorsName = ["black", "green", "blue", "red"]
+    private var colors = [UIColor.black, UIColor.systemGreen, UIColor.systemBlue, UIColor.systemRed]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        showViewElementandAction()
+    }
+    
+    private func showViewElementandAction() {
+        title = "Edit Text"
+        tabBarItem = UITabBarItem(tabBarSystemItem: .favorites, tag: 0)
+        tabBarItem.image = UIImage(named: "pencil")
+        view.addSubview(textLabel)
+        view.addSubview(linePicker)
+        view.addSubview(colorPicker)
+        view.addSubview(boldSwitch)
+        view.addSubview(textSizeSlider)
+        
+        textSizeSlider.addTarget(self, action: #selector(changeSliderAction), for: .valueChanged)
+        boldSwitch.addTarget(self, action: #selector(changeSwitchAction), for: .valueChanged)
+        linePicker.delegate = self
+        linePicker.dataSource = self
+        colorPicker.delegate = self
+        colorPicker.dataSource = self
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add,
+                                                            target: self, action: #selector(shoAlertAction))
+    }
+    @objc private func changeSwitchAction() {
+        if boldSwitch.isOn {
+            textLabel.font = .boldSystemFont(ofSize: 25)
+        } else {
+            textLabel.font = .systemFont(ofSize: 25)
+        }
+    }
+    @objc private func shoAlertAction() {
+        let alertController = UIAlertController(title: "Введите текст", message: "", preferredStyle: .alert)
+        alertController.addTextField()
+        let alertAddAction = UIAlertAction(title: "Добавить", style: .default) { _ in
+            guard let text = alertController.textFields?.first?.text else { return }
+            self.textLabel.text = text
+        }
+        let alertCancelAction = UIAlertAction(title: "Cancel", style: .cancel)
+        
+        alertController.addAction(alertAddAction)
+        alertController.addAction(alertCancelAction)
+        
+        present(alertController, animated: true)
+    }
+    @objc private func changeSliderAction() {
```

Comment:
```suggestion

    @objc private func changeSliderAction() {
```
отделяем 1 строкой методы друг от друга